server/schemas/userSchema.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
Re-enable this rule and fix this ESLint issue.

Reply:
Done that